content/XX.Peptide Fragmentation

```diff
@@ -0,0 +1,65 @@
+BeamCID typically uses nitrogen or argon as a collision gas, and peptide ions are accelerated in to a collision cell filled with several mTorr of bath gas.
+The kinetic energy used to accelerate precursor ions (often generated using direct current voltage differentials between the source of the ions and the collision cell) determines the energy imparted through collisions with the bath gas, which in turn governs their fragmentation behavior.
+In non-modified peptides, the most labile bonds are typically peptide bonds (i.e., amide bonds) between amino acids, so the increase in internal energy from beamCID generates b- and y-type ions that represent this peptide bond cleavage, as shown in Biemann fragment ion nomenclature (Figure X).
+b-type ions provide sequence information for fragments that have an intact N-terminus, while y-type ions denote fragment ions with an intact C-terminus.
+Collisions in beamCID typical impart enough internal energy to fragment peptide ions with a single collision, although the beam-type nature of rapidly moving ions into the collision cell can generate multiple collision events as ion traverse the device.
+Because the increase in internal energy happens rapidly before energy can be redistributed, beamCID can generate fragments that are not necessarily derived from cleavage of the most labile bonds (e.g., PTM-modified peptides, discussed below), but spectra are often dominated by b/y-type ions from amide bond cleavage.
+BeamCID can also generate secondary fragments, such as immonium ions from side chain losses [@DOI:10.1021/ac8006076] or a-type fragment ions that come from water loss from b-type ions due to multiple collision events (note: a-type ions can form as primary fragmentation products in other dissociation methods).
+The simplicity of beamCID, which requires simply an rf-only collision cell, has made it widely implemented on most instrument platforms used in modern proteomics.
+
+A second form of CID is called resonant CID (resCID), where the internal energy of peptide ions is slowly increased through multiple low-energy collisions [@DOI:10.1002/(SICI)1096-9888(199705)32:5<461::AID-JMS515>3.0.CO;2-H].
+Here, helium gas is most often used, as it imparts less energy per collision, and activation typically happens in ion trap devices where supplemental frequencies can be used to excite ions.
+In other words, ions are trapped using axial rf-frequencies, and an additional rf-frequency is applied to the electrodes of the ion trap [@DOI:10.1016/S1044-0305(02)00384-7; @DOI:10.1002/mas.21549].
+This supplemental rf is selected to have a frequency resonant with the fundamental frequency of the ions to be fragmented, as determined by the Matthieu equations, which excites the ions of interest so that they have increased kinetic energy as they move in the ion trap.
```

Comment:
```suggestion
This supplemental rf is selected to have a frequency resonant with the fundamental frequency of the ions to be fragmented, as determined by the Mathieu equations, which excites the ions of interest so that they have increased kinetic energy as they move in the ion trap [@DOI:10.1115/1.4039144; @DOI:10.1006/rwsp.2000.0143].
```